tests/console/validate_vnc_avail.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that after remote installation, the VNC connection
+# between client and host, is available without the use of vnc password.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use mmapi;
+use y2lan_restart_common "close_xterm";
+
+# Returns the first leased IP, in /var/lib/dhcp/db/dhcpd.leases file
```

Comment:
Could you comment why is needed the first?

Reply:
The truth is, it should be the only leased IP in the file, because the multi machine test includes only two machines (if I understand correctly). I just added the comment to make clear how the code will behave in different circumstances , in case someone else wants to use it. Should I remove the comment or change something?

Reply:
I still have doubts about this, because according to [this](https://manpages.debian.org/testing/isc-dhcp-server/dhcpd.leases.5.en.html) it should be the last for the last edition. The bigger problem is that this value is fixed in the support server, I think here were we set [the range](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/support_server/setup.pm#L211) so we might just use it as test data.

Reply:
Ok,  I removed the function for IP calculation and added the ip to test_data